File: my-folder/0001-two-sum/solution.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        Constraints:
        2 <= nums.length <= 103
        -109 <= nums[i] <= 109
        -109 <= target <= 109
        Only one valid answer exists.
        """
        nums = sorted([(idx, num) for (idx, num) in enumerate(nums)], key=lambda x: x[1])
        for e_i, (idx, num) in enumerate(nums):
            for (t_idx, t_num) in nums[e_i + 1:]:
                if target - num == t_num:
                    return [idx, t_idx]

File: my-folder/0001-two-sum/solution.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        // Write your code here
        Dictionary<int, int> indMap = new Dictionary<int, int>();
        for (int i = 0; i < nums.Count(); i++)
        {
            indMap[nums[i]] = i;
        }

        for (int i = 0; i < nums.Count(); i++)
        {
            int t = target - nums[i];

            if (indMap.ContainsKey(t) && indMap[t] != i)
            {
                return new int[]{ i, indMap[t] };
            }
        }
        return new int[0];
    }
}
